elide-contrib/elide-test-helpers/src/test/java/com/yahoo/elide/contrib/testhelpers/graphql/GraphQLDSLTest.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.testhelpers.graphql;
+
+import static com.yahoo.elide.contrib.testhelpers.graphql.GraphQLDSL.QUOTE_VALUE;
+import static com.yahoo.elide.contrib.testhelpers.graphql.GraphQLDSL.argument;
+import static com.yahoo.elide.contrib.testhelpers.graphql.GraphQLDSL.arguments;
+import static com.yahoo.elide.contrib.testhelpers.graphql.GraphQLDSL.document;
+import static com.yahoo.elide.contrib.testhelpers.graphql.GraphQLDSL.field;
+import static com.yahoo.elide.contrib.testhelpers.graphql.GraphQLDSL.query;
+import static com.yahoo.elide.contrib.testhelpers.graphql.GraphQLDSL.selection;
+import static com.yahoo.elide.contrib.testhelpers.graphql.GraphQLDSL.selections;
+import static com.yahoo.elide.contrib.testhelpers.graphql.GraphQLDSL.variableDefinition;
+import static com.yahoo.elide.contrib.testhelpers.graphql.GraphQLDSL.variableDefinitions;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class GraphQLDSLTest {
+
+    @Test
+    public void verifyBasicRequest() {
```

Comment:
Let's make sure the coverage of the new DSL is very high.  This should touch on every entry function you created in the DSL.